content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -62,6 +64,22 @@ This page shows you how to set up a simple Ingress which routes requests to Serv
     storage-provisioner                         1/1       Running   0          2m
     ```
 
+    For minikube  version 1.19.0 or later run
+
+    ```shell
+    kubectl get pods -n ingress-nginx
+    ```
+    {{< note >}}This can take up to a minute.{{< /note >}}
+
+     Output:
+
+    ```shell
```

Comment:
This should not be marked as shell; it's actually sample output. (It's OK to save fixing that for a separate PR).